CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add getter for path field in NestedQueryBuilder ([#4636](https://github.com/opensearch-project/OpenSearch/pull/4636))
 - Allow mmap to use new JDK-19 preview APIs in Apache Lucene 9.4+ ([#5151](https://github.com/opensearch-project/OpenSearch/pull/5151))
 - Add support for ppc64le architecture ([#5459](https://github.com/opensearch-project/OpenSearch/pull/5459))
+- Cancellation of in-flight SearchTasks based on resource consumption ([#5606](https://github.com/opensearch-project/OpenSearch/pull/5605))
```

Comment:
Will this go into the next 2.x release? If so, please move this to the unreleased 2.x section and make sure the appropriate labels get added to the PR.

Also, it's probably better to link to the issue (#5173), and can you rewrite this in the imperative mood? Maybe something like "Add backpressure cancellation of search tasks at the coordinator level"? But please feel free to rewrite that.

Reply:
Well yes, this should go in 2.x release, will move it to unreleased 2.x section and update labels.
Sure, will rewrite it as "Add search backpressure cancellation at the coordinator level".

Reply:
I've added the labels. I just mean put the link to the issue instead of the PR in the changelog. It has more relevant information for a user that wants to learn more about this change.

Reply:
Thanks. Sure, I will add the link to the issue.